engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3070,7 +3071,7 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
             }
 
             arr.put(oper);
-          } catch (Exception | NoClassDefFoundError ex) {
+          } catch (Exception | NoClassDefFoundError | UnsupportedClassVersionError ex) {
```

Comment:
Why add UnsupportedClassVersionError to the list if none of your code is throwing it?